packages/dashboard/src/components/iot-dashboard/iot-dashboard-widget/iot-dashboard-widget.tsx

```diff
@@ -11,10 +12,16 @@ export class IotDashboardWidget {
   @Prop() cellSize: number;
   @Prop() width: number;
   @Prop() widget: Widget;
+  @Prop() viewport: MinimalViewPortConfig;
 
   render() {
-    const { x, y, z, width, height, id } = this.widget;
+    const { x, y, z, width, height, id, componentTag, queries, properties, annotations } = this.widget;
     const { cellSize } = this;
+
+    // I18n?
```

Comment:
this should just be part of `messageOverrides` and handle all messaging in a consistent manner